.eslintrc.js

```diff
@@ -2,17 +2,16 @@
 module.exports = {
 	root: true,
 	plugins: ['import', 'jsdoc', 'regexp'],
-	extends: 'eslint:recommended',
 	env: {
 		es6: true,
 	},
 	parserOptions: {
 		ecmaVersion: 'latest',
-		sourceType: 'module'
+		sourceType: 'module',
 	},
 	rules: {
-		'no-use-before-define': ['error', { 'functions': false, 'classes': false }],
-		'eqeqeq': ['error', 'always', { 'null': 'ignore' }],
+		'no-use-before-define': ['error', { functions: false, classes: false }],
+		eqeqeq: ['error', 'always', { null: 'ignore' }],
```

Comment:
Do I get it right that quotes should be preserved here (and in some other places down there)? Or was this change made before we added the corresponding option to `.prettierrc`? Is there a chance that we should roll back all the changes and re-run prettier from scratch?

Reply:
Other than that, LGTM!

Reply:
Yes I will do that, great find